extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DatabaseDefaultSetupConfig.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.datasource.deployment.spi;
+
+public final class DatabaseDefaultSetupConfig {
+
+    private DatabaseDefaultSetupConfig() {
+    }
+
+    public static final String DEFAULT_DATABASE_USERNAME = "quarkus";
+
+    // mssql container enforces a 'strong' password with min 8 chars, upper/lowercase, number or special char
+    public static final String DEFAULT_DATABASE_PASSWORD = "quarkus1!";
```

Comment:
I think I would rather use `quarkus` for the password and have a `STRONG_DEFAULT_DATABASE_PASSWORD` for databases that requires it so only MSSQL for now.

Rationale is that in some cases you want to connect to the database while dev mode is running. And it's better to have a simple password if we can.